task4/script.js

```diff
@@ -0,0 +1,29 @@
+const ul = document.getElementById('ul');
+
+const array = [{
+    to: "bookmark.html",
+    img: "1.png", 
+    alt:"画像1",
+    text: "ブックマーク"
+}, 
+{
+    to: "message.html", 
+    img: "2.png", 
+    alt:"画像2", 
+    text: "メッセージ"
+}];
+
+for (let i =0; i <= array.length - 1; i++) {
```

Comment:
```suggestion
for (let i = 0; i < array.length; i++) {
```
[ IMO ]
I thought it would be simpler to write it this way.

Reply:
I made a mistake using the operator.
Thank you for your advice.